### PR TITLE
refactor(rxDiskSize): FRMW-372 move rxDiskSize filter to quarks

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-372

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxDiskSize` filter to quark construct.

### CODE REVIEW

**Comment by 100stacks** [comment]
## rxDiskSize filter refactored to Quark 

### Demo Tab
screen shot 2015-12-15 at 11 22 08 am

### Markup Tab
screen shot 2015-12-15 at 11 22 32 am

### JavaScript Tab
screen shot 2015-12-15 at 11 22 46 am

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/rxDiskSize/rxDiskSize.meta.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "rxDiskSize",
+    "stability": "stable",
+    "description": "Converts raw GB disk size into a more readable format",
+    "api": "filter:rxDiskSize",
```
Comment (100stacks) [inline src/quarks/rxDiskSize/rxDiskSize.meta.json]: Added for API documentation routing.

File: src/quarks/rxDiskSize/docs/examples/rxDiskSize.demo.html [hunk 2]
```diff
@@ -1,5 +1,4 @@
-<!-- Sample HTML goes here as a live example of how to the component can be used -->
-<div ng-controller="rxDiskSizeCtrl">
+<div ng-controller="rxDiskSizeCtrl" id="rx-disk-size-demo">
     <ul>
         <li>{{sizeGB}} &rarr; {{sizeGB | rxDiskSize}}</li>
         <li>{{sizeTB}} &rarr; {{sizeTB | rxDiskSize}}</li>
```
Comment (CITguy) [inline src/quarks/rxDiskSize/docs/examples/rxDiskSize.demo.html]: I'm not a fan of file naming, but this will be addressed when refactoring examples.

Comment (CITguy) [inline src/quarks/rxDiskSize/docs/examples/rxDiskSize.demo.html]: FYI: https://jira.rax.io/browse/FRMW-555 created for refactoring.

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**